wit/batch.wit

```diff
@@ -0,0 +1,64 @@
+/// A keyvalue interface that provides batch operations.
+/// 
+/// A batch operation is an operation that operates on multiple keys at once.
+/// 
+/// Batch operations are useful for reducing network round-trip time. For example, if you want to
+/// get the values associated with 100 keys, you can either do 100 get operations or you can do 1
+/// batch get operation. The batch operation is faster because it only needs to make 1 network call
+/// instead of 100.
+/// 
+/// A batch operation does not guarantee atomicity, meaning that if the batch operation fails, some
+/// of the keys may have been modified and some may not. 
+/// 
+/// This interface does has the same consistency guarantees as the `store` interface, meaning that
+/// you should be able to "read your writes."
+interface batch {
+    use types.{bucket, error};
+
+    /// The batch resource contains functions for fetching or updating many keys at once.
+    resource batch {
```

Comment:
Same comment as i made on `atomics` above.

Reply:
So I went to make a change and everything looks weird to me. Should this be `batches`? That looks a little odd. `batchers` I don't think is a word 😅 `batch` can be both a noun and a verb so maybe that still works? Maybe I've been staring at this too long

Reply:
Oh I think Luke was originally suggesting that the `batch` operations could be defined as extension to `bucket` just like the commen he made on atomics interface.

Reply:
Ah ok. Will resolve this